classes/persistent/sitenotice.php

```diff
@@ -63,6 +63,11 @@ protected static function define_properties() {
                 'null' => NULL_NOT_ALLOWED,
                 'default' => 0,
             ],
+            'forcelogout' => [
```

Comment:
we probably also need to add this to a UI table where we display all notices.

Reply:
Oops, forgot that one. Done.